evalml/tests/dependency_update_check/minimum_requirements.txt

```diff
@@ -1,4 +1,4 @@
-plotly==5.0.0
+plotly==4.14.0
```

Comment:
Wait... how did the deps update bot choose to lower a min version?? I thought it would only modify the latest requirement versions. I could be misunderstanding though. @gsheni do you happen to know the answer? Tagging @freddyaboulton too

Reply:
I am unsure why this happened, but I imagine its similar reason as above. It is the correct version, based on whats in `main` correctly.

Reply:
@chukarsten had to modify our unit tests to be compatible with plotly 5.0.0. So I think what we should do is change `requirements.txt` to be `plotly>=5.0.0` and disregard this change. 

https://github.com/alteryx/evalml/pull/2415/files